Bookish/Views/Shared/_CopyTable.cshtml

```diff
@@ -7,25 +7,102 @@
             <th scope="col">Id</th>
             <th scope="col">Borrower</th>
             <th scope="col">Due date</th>
+            <th scope="col">Overdue by</th>
+            <th></th>
+            <th></th>
         </tr>
         </thead>
         <tbody>
         @foreach (var copy in Model)
         {
-            <tr>
+            <tr class="@(!copy.IsAvailable() && copy.OverdueTime().Days >= 1 ? "table-danger" : "")">
+                
                 <th scope="row">@copy.Id</th>
+                
                 @if (!copy.IsAvailable())
                 {
                     <td>@copy.Borrower.Name</td>
                     <td>@copy.DueDate</td>
+                    @if (copy.OverdueTime().Days >= 1)
+                    {
+                        <td>@copy.OverdueTime().Days days</td>
+                    }
+                    else
+                    {
+                        <td></td>
+                    }

```

Comment:
You could probably reduce this to one line using a ternary and string interpolation, something like:
```
<td>@(copy.OverdueTime().Days >= 1 ? $"{copy.OverdueTime().Days} days" : "")</td>
```